src/main/resources/openapi/PrintServiceAPI.yaml

```diff
@@ -0,0 +1,155 @@
+openapi: 3.0.0
+info:
+  title: Print Service API
+  version: '1.0'
```

Comment:
We are using semantic versioning for openAPI specs (and we are actively maintaining the version number on any and all updates to a spec). Please can you set this to `1.0.0` please?

Reply:
I'm not sure that this should be considered the `1.0.0` version of the spec - I assume this to be a "starter-for-ten" spec that will change once the features are refined and requirements become clearer.

I would prefer that the spec in `main/resources/api` are maintained by the developers when the API is developed, this ensures that it goes through the PR process when requirements are understood by the team.  Additionally, only endpoints that are actually implemented appear in this specification.

Reply:
Good shout Neil, perhaps this should be in a `/migration' directory off of the top level (and not part of the src tree), with the filename being explicit that its is a starter for 10. Much like what we are proposing in notifications-api:
https://github.com/cabinetoffice/eip-ero-notifications-api/blob/feature/EIP1-557-initial-openapi-spec-for-sqs-messages/migration/starter-for-10-NotificationsAPIs.yaml